app/controllers/search_controller.rb

```diff
@@ -0,0 +1,11 @@
+class SearchController < ApplicationController
+  def search_query
+    @query = params[:query]
+    @users = @posts = @communities = []
+    if @query != ""
```

Comment:
Par to kadreiz bija saruna, kā salidzinošana iekš Ruby ar != un == var izsaukt coercion (implicit conversion)

if query.blank?